test/src/unit-capi-array.cc

```diff
@@ -699,6 +700,387 @@ TEST_CASE_METHOD(
   }
 }
 
+TEST_CASE_METHOD(
+    ArrayFx,
+    "C API: Test array with encryption, no with_key API",
```

Comment:
No need to make this specific to `no with_key API`. The entire code and tests must remove all occurrences of the deprecated APIs, and use the keys in config as the "standard" way.